parsec/core/gui/main_window.py

```diff
@@ -553,19 +562,21 @@ def add_instance(self, start_arg: Optional[str] = None):
             except ValueError as exc:
                 show_error(self, _("TEXT_INVALID_URL"), exception=exc)
 
-        idx = self._get_login_tab_index()
-        if idx != -1:
-            self.switch_to_tab(idx)
-        else:
-            tab = self.add_new_tab()
-            tab.show_login_widget()
-            self.on_tab_state_changed(tab, "login")
-            self.switch_to_tab(self.tab_center.count() - 1)
-            idx = self.tab_center.count() - 1
+        # Do not open a new logging widget if the organisation is already opened
```

Comment:
Not sure about the comment here: if we don't go into the if we open the logging widget, and if we go into the if we call a function that maybe open the logging widget !

So I guess this comment would be better within the `go_to_file_link` function ?

Reply:
removed, as you mentioned, this is already handled in `go_to_file_link`